app/assets/javascripts/application/suggest_user.js

```diff
@@ -0,0 +1,24 @@
+function suggestUser() {
+    var url = "/search/suggest_leader";
+    var params = "query=" + document.getElementsByName('group_leader_name')[0].value;
```

Comment:
Let's use const here instead of var, and also string templates.

Reply:
Done!